docs/kpi-options/references.md

```diff
@@ -9,6 +9,9 @@ sidebar_label: References
     - [UMA’s TVL Options experiment](https://medium.com/uma-project/uma-kpi-options-and-airdrop-bae86be16ce4)
 - Implementations:
     - [BadgerDAO’s “Rebase Mining” Implementation](https://medium.com/uma-project/badgerdaos-rebase-mining-kicks-off-now-using-uma-s-kpi-options-cd75f71dc1fa)
+    - [GYSR KPI Options Integration](https://medium.com/uma-project/learn-about-gysr-streamed-kpi-options-41ac26cbc613)
+    - [QiDao Strategic Partnership Program](https://0xlaozi.medium.com/qidao-partners-with-uma-to-power-strategic-partner-program-cfd0d29adbf6)
+    - [Redacted KPI Options Documentation](https://redacted-cartel.gitbook.io/redacted/ecosystem-alignements/uma)
```

Comment:
can we add wagmi as well?
- https://boba.network/wagmi/
- https://medium.com/uma-project/boba-network-partners-with-uma-to-create-wagmi-kpi-options-d967427ebad6

Reply:
Added both of these.